insalan/tournament/views.py

```diff
@@ -176,12 +173,12 @@ def get(self, request, primary_key: int):
             ).data
             del team_preser["tournament"]
 
-            # Dereference players/managers to users (username)
```

Comment:
I'm not entirely sure we should be dereferencing registrations to the nicknames here. We never clearly defined the invariants on those structures, but if I were the front-end I'd expect that what the backend gives me is usernames (maybe *alongside* nicknames, like in pairs/tuples), because I might use those to pull additional information later

Reply:
Agreed, it would also be way more convenient later if we want to show profile pictures, etc.

Reply:
You're right, I did this when I was only focused on teams display on the fronted. I think a tuple with (user id, nickname) would be a good solution, what do you think ?